lm_eval/evaluator.py

```diff
@@ -129,25 +133,30 @@ def simple_evaluate(
             if task_obj is None:
                 continue
 
-        config = task_obj._config
-        if config["output_type"] == "generate_until" and gen_kwargs is not None:
-            config["generation_kwargs"].update(gen_kwargs)
+        if task_obj.get_config("output_type") == "generate_until":
+            if gen_kwargs is not None:
+                task_obj.override_config(
```

Comment:
Seems like source for fails, when task is an instance of Task with no override_config available